client/src/pages/chat/Chat.jsx

```diff
@@ -0,0 +1,113 @@
+import "./chat.css";
+import { useState } from "react";
+import Navbar from "../../components/navbar/Navbar.jsx";
+import CloseSessionBox from "../../components/chatComponents/chatPrescriptions/closeSessionBox/CloseSessionBox.jsx";
+import SelectChat from "../../components/chatComponents/selectChat/SelectChat.jsx";
+import ChatHead from "../../components/chatComponents/chatHead/ChatHead.jsx";
+import TypingBar from "../../components/chatComponents/typingBar/TypingBar.jsx";
+import DisplayMessages from "../../components/chatComponents/displayMessages/DisplayMessages.jsx";
+import ChatPrescriptions from "../../components/chatComponents/chatPrescriptions/ChatPrescriptions.jsx";
+import AddPrescription from "../../components/chatComponents/chatPrescriptions/addPrescription/AddPrescription.jsx";
+
+const DisplayType = Object.freeze({
+  DOCTOR: 1,
+  PATIENT: 2,
+});
+
```

Comment:
Used in multiple places maybe you can factor it out into a global constants file or something similar.

Reply:
I think it is used in two places only. So ill implement it later on if the list increases.